.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+Style/Encoding:
+  Enabled: false
```

Comment:
For all cops we change, let's have a comment in the rubocop.yml that explains the reasoning.